pom.xml

```diff
@@ -44,24 +44,24 @@
 
     <buildnumber.maven.plugin.version>1.4</buildnumber.maven.plugin.version>
     <coveralls.maven.plugin.version>4.3.0</coveralls.maven.plugin.version>
-    <eg.oss.plugin.config.version>1.1.0</eg.oss.plugin.config.version>
-    <failsafe.maven.plugin.version>2.22.0</failsafe.maven.plugin.version>
-    <jacoco.version>0.8.4</jacoco.version>
+    <eg.oss.plugin.config.version>1.2.0-SNAPSHOT</eg.oss.plugin.config.version>
```

Comment:
OK, let's leave this like this until this is released as you pointed out. Then we can change this to 1.2.0 and then do the merge.